models/opensea/ethereum/opensea_v4_ethereum_events.sql

```diff
@@ -1,558 +1,45 @@
 {{ config(
     schema = 'opensea_v4_ethereum',
     alias = alias('events'),
-    partition_by = ['block_date'],
+    tags = ['dunesql'],
     materialized = 'incremental',
     file_format = 'delta',
     incremental_strategy = 'merge',
     unique_key = ['block_date', 'unique_trade_id']
```

Comment:
looks like unique keys differ here compared to arbitrum version. is that temporary until the field is fully deprecated later?